### PR TITLE
improve: Log package version on startup

### PR DESCRIPTION
**Pull request by pxrl**
Could alternatively be bundled into the global config within src/common/Config.ts, so it would be automagically logged by the respective "{Dataworker,Relayer} started" log messages.

### CODE REVIEW

**Review by nicholaspai — COMMENTED** [review]
I like logging this in the global config since I'd want to know the version for the dataworker run as well!

**Comment by pxrl** [comment]
> I like logging this in the global config since I'd want to know the version for the dataworker run as well!

This logs unconditionally - so it is always available, no matter how the bot is started. It generates an additional log message, though.

I'll move it to global config.

File: index.ts [hunk 1]
```diff
@@ -46,6 +47,8 @@ export async function run(args: { [k: string]: boolean | string }): Promise<void
 }
 
 if (require.main === module) {
+  // Inject version into process.env so CommonConfig and all subclasses inherit it.
+  process.env.VERSION = version;
```
Comment (pxrl) [inline index.ts:51]: Hm, VERSION might be a bit too generic. Should probably prefix by something Across-specific.

**Comment by pxrl** [comment]
FWIW one additional thing that I think we need is to make the git commit hash available to the bot, so it can be logged as well. That's a little bit more complex to implement though.

**Review by nicholaspai — APPROVED** [review]

**Pull request merged**